testcafe/tests/drive/clean-data.js

```diff
@@ -0,0 +1,52 @@
+import { FOLDER_NAME } from '../helpers/data'
+import { checkToastAppearsAndDisappears } from '../pages/commons'
+import { driveUser } from '../helpers/roles'
+//import { Role } from 'testcafe'
+import { TESTCAFE_DRIVE_URL, SLUG } from '../helpers/utils'
+import { initVR } from '../helpers/visualreview-utils'
+let data = require('../helpers/data')
```

Comment:
Why are we using a `let … require` instead of an `import` here?

Reply:
change back to import